test/test_fast.rb

```diff
@@ -206,8 +207,7 @@ def test_move_relative
        ['/array/1/hash', 'h2/a', '/array/1/hash/h2/a'],
        ['/array/1', 'hash/h2/a/2', '/array/1/hash/h2/a/2'],
        ['/array/1/hash', '../string', '/array/1/string'],
-       ['/array/1/hash', '..', '/array/1'],
-      ].each do |start, path, where|
+       ['/array/1/hash', '..', '/array/1'],].each do |start, path, where|
```

Comment:
Probably need to manually adjust to prevent folding

Reply:
Better the original way.